drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -3640,6 +3640,46 @@ void RasterizerStorageGLES3::mesh_add_surface(RID p_mesh, uint32_t p_format, VS:
 		surface->attribs[i] = attribs[i];
 	}
 
+	{
```

Comment:
I suggest storing the AABB on CPU side, and then create the vertex buffer on demand the first time it is needed. Otherwise this runs for every object created even if occlusion culling is never used

Reply:
I wanted to avoid doing stuff on demand but maybe you are right.

Reply:
I agree that creating buffers on demand sucks. But we can't add a fixed cost to every single project just to benefit users of a particular feature